sources/testme/js/kTestMe.js

```diff
@@ -270,7 +270,18 @@ var kTestMe = {
 		
 		this.classes[type].subClasses = subTypes;
 		for(var i = 0; i < subTypes.length; i++)
+		{
 			this.registerClass(subTypes[i]);
+			
+			if(subTypes[i].hasOwnProperty("isArray") && subTypes[i].isArray && subTypes[i].hasOwnProperty("arrayType") && subTypes[i].arrayType.hasOwnProperty("properties"))
```

Comment:
You can assume that the type contains isArray, arrayType, isComplexType and properties, no need to check their existence.
Also, check that registering the properties classes wont cause any damage when the class is abstract and need its sub-classes to be loaded.